package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.3",
   "description": "Split a polygon using a polyline.",
   "main": "dist/polygonsplitter.js",
-  "module": "src/polygonsplitter.es.js",
+  "module": "dist/polygonsplitter.mjs",
```

Comment:
You might also need a block like this, but I'm not 100% on why. The main turf packages needed this but it might have been because we weren't publishing mjs extension.

```
  "exports": {
    "./package.json": "./package.json",
    ".": {
      "import": "dist/polygonsplitter.mjs",
      "require": "dist/polygonsplitter.js"
    }
  },
  ```